SNR/hdf5_rtplot_utils.py

```diff
@@ -78,34 +78,40 @@ def plot_unaveraged_range_time_data(data_array, num_sequences_array,
     start_sample
         The sample to start plotting at. 
     end_sample
-        The last sample in the sequence to plot. 
+        The last sample in the sequence to plot.
+    figsize
+        The desired size (in inches) of the plotted figure.
     """
     print(dataset_descriptor)
     (num_records, max_num_sequences, num_samps) = data_array.shape
 
-    power_list = [] # list of lists of power
-    timestamps = [] # list of timestamps
-    noise_list = [] # list of (average of ten weakest ranges in sample range)
-    max_snr_list = [] # max power - sequence noise (ave of 10 weakest ranges)
+    power_list = []     # list of lists of power
+    timestamps = []     # list of timestamps
+    noise_list = []     # list of (average of ten weakest ranges in sample range)
+    max_snr_list = []   # max power - sequence noise (ave of 10 weakest ranges)
+
     for record_num in range(num_records):
         num_sequences = int(num_sequences_array[record_num])
-        # get all antennas, up to num sequences, all samples for this record.
-        voltage_samples = data_array[record_num,:num_sequences,:]
+
+        # get data for all sequences up to num sequences for this record.
+        voltage_samples = data_array[record_num, :num_sequences, :]
 
         for sequence in range(num_sequences):
-            timestamp = float(timestamps_array[record_num, 
-                                                       sequence])
-            # power only. no averaging done. 
-            power = np.sqrt(voltage_samples.real**2 + 
-                            voltage_samples.imag**2)[sequence,
-                            start_sample:end_sample]
+            timestamp = float(timestamps_array[record_num, sequence])
+            timestamps.append(float(timestamp))
```

Comment:
```suggestion
            timestamps.append(timestamp)
```
It's cast to a float in the line above